skimage/color/colorconv.py

```diff
@@ -842,6 +842,12 @@ def gray2rgb(image, alpha=None):
             is_rgb = True
 
     if is_rgb:
+        warn('Pass-through of possibly RGB images in gray2rgb is deprecated. '
```

Comment:
I think the appropriate parameter to add is one called multichannel

Reply:
`multichannel` is in fact well suited for this, but we all agree that in version 0.19 this parameter will be deprecated since nD will be the default behavior of gray2rgb?

Reply:
Yeah..... Maybe the best way is to ask users to import the function from a "future" module???

Definitely annoying

Reply:
Not sure the best way to go about this one

Reply:
So, one thing I've been thinking about is to add a note to the next two release notes, 0.17 and 0.18, that users should pin to <1.0, because 1.0 will contain breaking changes, and then work to get 1.0 out after that, with a bunch of breaking changes. This would solve this situation and a bunch of others... But probably I should start a SKIP for this. =)

Reply:
I'm ok with this as is.

Users that avoid this warning will have code that is correct all the way through